pkg/cmd/pr/create/create.go

```diff
@@ -206,6 +208,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.Bool("no-maintainer-edit", false, "Disable maintainer's ability to modify pull request")
 	fl.StringVar(&opts.RecoverFile, "recover", "", "Recover input from a failed run of create")
 	fl.StringVarP(&opts.Template, "template", "T", "", "Template `file` to use as starting body text")
+	fl.BoolVar(&opts.PushToBaseRepo, "push-to-base-repo", false, "Push the head branch to the base repository")
```

Comment:
If we're going to expand flags to allow people to determine this from the command line, knowing there are more options that people want, then I imagine this might be more generic flag with different values for `prompt`, `base`, ...

Reply:
Interesting idea! I took a look at the code, but I couldn't find any other 'canonical' repository names except for `base` (which already has an implementation for that we're using).

For `prompt`, that's already the default behaviour if this flag isn't passed

https://github.com/cli/cli/blob/363dacbbed8cc9f86026ec1b1e38c6dca0d78c08/pkg/cmd/pr/create/create.go#L577

Are you suggesting accepting any value here and validating it against the `pushableRepos` as seen on https://github.com/arturhoo/cli/compare/push-to-base-repo...arturhoo:cli:push-to-repo?expand=1 ?